### PR TITLE
change deploy file names and add concept docs

### PR DESCRIPTION
**Pull request by gomathyk**

### CODE REVIEW

File: en/docs/concepts/claim-management.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    .
+
+A set of claims is identified as a dialect. Different dialects represent
+the same piece of information with different claim URIs.
+
+With the help of the Claim Management feature, different claims can be
+mapped with the attributes of the user that are associated with their
+profile.
+
+You can do the following tasks with the Claim Management feature.
```
Comment (sherenewso2) [inline en/docs/concepts/claim-management.md:56]: this line should not be in the concepts section

File: en/docs/concepts/consent-management.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+WSO2 IS can be used to manage consents that belong to third party
+applications and services. To do this, third party applications can
+integrate with WSO2 IS using the consent REST APIs and other admin APIs
+provided by WSO2 IS. Note that generating and handling consent UIs for
+third party applications can not be supported in WSO2 IS as it is
+considered the responsibility of the relevant third party application to
+provide consent management UI facilities for their end users.
+
+!!! tip For more information and instructions for setting up this use
```
Comment (sherenewso2) [inline en/docs/concepts/consent-management.md:172]: check formatting of tip

File: en/docs/concepts/identity-anti-patterns-and-the-identity-bus.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    back to the service provider from the identity bus.
+6.  **Just-in-time provisioning.** Since identity bus is at the middle
+    of all identity transactions - it can provision all external user
+    identities to an internal user store.
+7.  **Centralized monitoring and auditing.**
+8.  **Introducing a new federation protocol needs minimal changes.** If
+    you have a service provider or an identity provider, which supports
+    a proprietary federation protocol, then you only need to add that
+    capability to the identity bus. No need to implement it at each and
+    every identity provider or service provider.
```
Comment (sherenewso2) [inline en/docs/concepts/identity-anti-patterns-and-the-identity-bus.md:116]: At the end of each concept page, we will have to link to the relevant guides. Better to track this as a task in our plan

File: en/docs/deploy/add-logs-for-tokens.md [hunk 4]
```diff
@@ -9,16 +9,16 @@ disabled from appearing in the logs by setting each token to
...
+UserClaims=true
+UserIdToken=true
+XACML_Request=true
+XACML_Response=true
+NTLM_Token=true
+SAML_Assertion=true
+SAML_Request=true
+AccessToken=true
+RefreshToken=true
+AuthorizationCode=true
```
Comment (sherenewso2) [inline en/docs/deploy/add-logs-for-tokens.md:21]: does the + icon show up as a bullet point? Just checking

File: en/docs/deploy/configure-a-read-only-ldap-user-store.md [hunk 5]
```diff
@@ -59,7 +59,7 @@ Following are the minimum userstore properties that are needed to be provided to
 <br />
 If you are connecting over ldaps (secured LDAP)<br />
 Need to import the certificate of userstore to the client-truststore.jks of the WSO2 product. For information on how to add certificates to the truststore and how keystores are configured and used in a system, see Using Asymmetric Encryption.<br />
-<a href="../../../deploy/security/using-asymmetric-encryption">Using asymmetric encryption</a><br />
+<a href="../../../deploy/security/use-asymmetric-encryption-basic">Using asymmetric encryption</a><br />
```
Comment (sherenewso2) [inline en/docs/deploy/configure-a-read-only-ldap-user-store.md:62]: why is it named basic? we don't have an advanced version right?

File: en/mkdocs.yml [hunk 6]
```diff
@@ -416,69 +416,69 @@ nav:
...
         - 'Network-Level': deploy/security/network-level-security-guidelines.md
       - 'Encryption':
         - 'Asymmetric Encryption':
-          - 'Use Asymmetric Encryption': deploy/security/using-asymmetric-encryption.md
-          - 'Create New Keystores': deploy/security/creating-new-keystores.md
-          - 'Configure Keystores': deploy/security/configuring-keystores-in-wso2-products.md
-          - 'Renew a CA-Signed Certificate in a Keystore': deploy/security/renewing-a-ca-signed-certificate-in-a-keystore.md
-          - 'Manage Keystores via UI': deploy/security/managing-keystores-with-the-ui.md
-          - 'Add Multiple Keys to the Primary Keystore': deploy/security/adding-multiple-keys-to-the-primary-keystore.md
+          - 'Use Asymmetric Encryption': deploy/security/use-asymmetric-encryption-basic.md
```
Comment (sherenewso2) [inline en/mkdocs.yml:455]:
```suggestion
          - 'Use Asymmetric Encryption': deploy/security/use-asymmetric-encryption.md
```

File: en/mkdocs.yml [hunk 7]
```diff
@@ -416,69 +416,69 @@ nav:
...
+          - 'Create New Keystores': deploy/security/create-new-keystores.md
+          - 'Configure Keystores': deploy/security/configure-keystores-in-wso2-products.md
+          - 'Renew a CA-Signed Certificate in a Keystore': deploy/security/renew-a-ca-signed-certificate-in-a-keystore.md
+          - 'Manage Keystores via UI': deploy/security/manage-keystores-with-the-ui.md
+          - 'Add Multiple Keys to the Primary Keystore': deploy/security/add-multiple-keys-to-the-primary-keystore.md
         - 'Symmetric Encryption':
           - 'Overview': deploy/security/symmetric-overview.md
           - 'Configurations Related to Symmetric Key Encryption': deploy/security/using-symmetric-encryption.md
```
Comment (sherenewso2) [inline en/mkdocs.yml:463]: this should change to use-symmetric-encryption.md

File: en/mkdocs.yml [hunk 8]
```diff
@@ -352,62 +352,62 @@ nav:
...
-        - 'Change to remote H2': deploy/changing-to-remote-h2.md
+        - 'Change to IBM DB2': deploy/change-to-ibm-db2.md
+        - 'Change to MariaDB': deploy/change-to-mariadb.md
+        - 'Change to MSSQL': deploy/change-to-mssql.md
+        - 'Change to MySQL': deploy/change-to-mysql.md
+        - 'Change to Oracle': deploy/change-to-oracle.md
+        - 'Change to Oracle RAC': deploy/change-to-oracle-rac.md
+        - 'Change to PostgreSQL': deploy/change-to-postgresql.md
+        - 'Change to remote H2': deploy/change-to-remote-h2.md
         - 'Change the Default Datasource of BPS': deploy/changing-the-default-datasource-for-bps.md
```
Comment (sherenewso2) [inline en/mkdocs.yml:376]: the file name should change to "change-the-default..."

File: en/mkdocs.yml [hunk 9]
```diff
@@ -416,69 +416,69 @@ nav:
...
       - 'Minikube': deploy/deployment-options/minikube.md
     - 'Deployment Patterns': deploy/deployment-guide.md
-    - 'Set up WSO2 clusters with Nginx': deploy/fronting-with-the-nginx-load-balancer.md
+    - 'Set up WSO2 clusters with Nginx': deploy/front-with-the-nginx-load-balancer.md
     - 'Set up Separate Databases for Clustering': deploy/settinng-up-separate-databases-for-clustering.md
-    - 'Change the hostname': deploy/changing-the-hostname.md
+    - 'Change the hostname': deploy/change-the-hostname.md
     - 'Introduction to User Management': deploy/introduction-to-user-management.md
   - 'APIs':
     - 'Using APIs': apis/using-apis.md
```
Comment (sherenewso2) [inline en/mkdocs.yml:484]: There are one or two file names under APIs section that needs to change as well (retrieving-tenant-resources-based-on-search-parameters.md, calling-admin-services.md, using-configuration...") please check

File: en/docs/deploy/add-logs-for-tokens.md (hunk 4 above)
Comment (gomathyk) [reply en/docs/deploy/add-logs-for-tokens.md:21]: These aren't plus icons that I added. It's just github showing the latest change.

File: en/docs/concepts/identity-anti-patterns-and-the-identity-bus.md (hunk 3 above)
Comment (gomathyk) [reply en/docs/concepts/identity-anti-patterns-and-the-identity-bus.md:116]: Added to the sheet

File: en/docs/concepts/consent-management.md (hunk 2 above)
Comment (gomathyk) [reply en/docs/concepts/consent-management.md:171]: Appears properly in the local

**Review by sherenewso2 — APPROVED** [review]

**Pull request merged**